docs/api/components.md

```diff
@@ -235,11 +235,17 @@ angular.component('myComponent', {
   controller: connect({
     foo: state`foo`,
     click: signal`clicked`
-  }, function MyController () {
+  }, 'myComponent', function MyController () {
     // Optionally add custom behaviour to controller
   })
 })
 ```
+Since angular doesn't expose the component name,
+you will need to provide one to `connect` for the
+component to be given a name in cerebral.
+
+Both component name and controller are optional arguments.
+You can provide either or both, and in any order.
```

Comment:
@christianalfoni and me landed on this after some discussion (before this PR was created), but for me it doesn't matter much, changing it is fine with me. Depends on whether we want to force users to create a name when creating a controller...